idunn/api/places_list.py

```diff
@@ -103,7 +112,15 @@ def valid_category(cls, v):
 
 
 class EventQueryParam(CommonQueryParam):
-    pass
+    outing_types: str = None
```

Comment:
why plural ? I think a single value is expected here.

Also, this parameter name is too specific in my opinion. On reflection, using `type` may be confusing, because `"type":"event"` is already present in the response. I would stick to `category`, similarly to the /places endpoint.

Reply:
ok I will redefine it as 'outing_category'